spec/services/idv/agent_spec.rb

```diff
@@ -15,6 +15,7 @@
     let(:issuer) { 'fake-issuer' }
     let(:friendly_name) { 'fake-name' }
     let(:app_id) { 'fake-app-id' }
+    let(:ipp_enrollment_in_progress) { true }
```

Comment:
Some of these specs use `MOCK_IDV_APPLICANT` and `MOCK_IDV_APPLICANT_WITH_SSN` - those are reflective of a remote applicant and those would need `ipp_enrollment_in_progress` to be `false`. So I think we need to update this value to `false` in the specs where it's appropriate.

But also I think we might need to update which mock applicant is used in some of the specs in this file. Not sure if that needs to happen in this pr.

Reply:
Oh good catch! Yeah I set this at the top level in progressive proofer, and removed it there. So I'll do that here too.